sandbox-backend/src/main/java/com/exadel/sandbox/team5/service/impl/EmployeeServiceImpl.java

```diff
@@ -59,4 +63,25 @@ public void removeSubscriptions(Long employeeId, Set<Long> categoryIds) {
         employee.getSubscriptions().removeAll(categoryDAO.getCategoryByIdIsIn(categoryIds));
         employeeDAO.save(employee);
     }
+
+    @Override
+    public void addFavorite(Long id) {
+        var employee = employeeDAO.getByLogin(SecurityUtils.getCurrentUsername());
+        employee.getFavorites().add(discountDAO.getById(id));
```

Comment:
In such cases it is better to save favorite separately without loading employee then you will have two db query instead of several